src/devices/cpu/mpk1839/kl1839vm1.h

```diff
@@ -0,0 +1,94 @@
+// license:BSD-3-Clause
+// copyright-holders:Andrei I. Holub
+
+#ifndef MAME_CPU_MPK1839_KL1839VM1_H
+#define MAME_CPU_MPK1839_KL1839VM1_H
+
+#pragma once
+
+
+enum
+{
+	KL1839_MCA = STATE_GENPC, KL1839_VMA, KL1839_IF, KL1839_RSP, KL1839_PCM, KL1839_RC, KL1839_RV, KL1839_SCH,
+	VAX_R0, VAX_R1, VAX_R2, VAX_R3, VAX_R4, VAX_R5, VAX_R6, VAX_R7, VAX_R8, VAX_R9, VAX_RA, VAX_RB,
+	VAX_AP, VAX_FP, VAX_SP, VAX_PC,
+	VAX_AK0, VAX_AK1, VAX_AK2, VAX_AK3, VAX_AK4, VAX_AK5, VAX_AK6, VAX_AK7, VAX_AK8,
+	VAX_RNK, VAX_RKA, VAX_PSL, VAX_BO
+};
```

Comment:
If you want drivers to have access to this register list, a better place for new CPU emulators is under public:.

Reply:
Not sure how it's going to be atm. Suspecion is that drivers has no acess to this registers directly. User code is interprited by microcode of cpu.